code/game/objects/items/grenades/discogrenade.dm

```diff
@@ -108,6 +108,10 @@
 		M.show_message("<span class='warning'>You resist your inner urges to break out your best moves.</span>", 2)
 		M.set_drugginess(5)
 		return
+	if(istype(M.get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/foilhat))
+		to_chat(M, "<span class = 'userdanger'>THOSE GLOW-IN-THE-DARK NANOTRASEN LIGHTBULBS WON'T CORRUPT ME WITH THEIR QUEER AGENDA</span>")
```

Comment:
> QUEER AGENDA

Reply:
yes.

Reply:
No

Reply:
You got anything better

Reply:
Xenophilic?

Reply:
i dunnoooo....

Reply:
Shouldn't be any politics in ss13 and if there is then everything should be allowed.

Reply:
Lmao queer agenda